commands/meshuggah.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = function meshuggah (controller) {
+  // Serve up a Meshuggah video
+  controller.hears('Meshuggah', params.triggerEvents, function(bot,message) {
+    Scrape.request('https://www.youtube.com/user/Meshuggah', function (err, $) {
```

Comment:
Woops, copy/paste errors. I dropped this in from my own Slackbot murfbot last night. Will take a look if I have some tinkering time this wknd. Thx Cody